cumulus/parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -468,11 +468,14 @@ construct_runtime!(
 		PolkadotXcm: pallet_xcm::{Pallet, Call, Event<T>, Origin, Config<T>} = 31,
 		CumulusXcm: cumulus_pallet_xcm::{Pallet, Event<T>, Origin} = 32,
 		DmpQueue: cumulus_pallet_dmp_queue::{Pallet, Call, Storage, Event<T>} = 33,
-		MessageQueue: pallet_message_queue::{Pallet, Call, Storage, Event<T>} = 34,
 
 		// Handy utilities.
 		Utility: pallet_utility::{Pallet, Call, Event} = 40,
 		Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 41,
+
+		// Message Queue. Importantly, is registered last so that messages are processed
+		// after the `on_initialize` hooks of bridging pallets (snowbridge in particular).
+		MessageQueue: pallet_message_queue::{Pallet, Call, Storage, Event<T>} = 99,
```

Comment:
```suggestion
		MessageQueue: pallet_message_queue::{Pallet, Call, Storage, Event<T>} = 250,
```
Then we still have room at the end and dont need to move the MQ backwards again.

Reply:
Good point! Fixed in https://github.com/paritytech/polkadot-sdk/pull/2230/commits/91265507fe7e790fac20eea2bbd952d671c332f7.